src/server/systems/ServerGameController.hpp

```diff
@@ -32,10 +35,12 @@ class ServerGameController : public GameController {
 
     // Methods
     void setNetworkState();
+    void addEventListeners();
 
     // Variables
     std::unique_ptr<NetworkingServer> m_networkingServer;
     TerminalRenderingController m_terminalRenderingController;
+    GameStateEvent m_gameStateEvent;
```

Comment:
What are the default values for field in GameStateEvents? Wondering if we need to 0 it out after creation to avoid random values

Reply:
```
struct GameStateEvent {
    std::vector<Collision> collisions;
};
```

A vectors default value is just an empty list